app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -106,6 +106,7 @@ public void onReceive(Context context, Intent intent) {
     private final BroadcastReceiver showUpdateDialog = new BroadcastReceiver() {
         @Override
         public void onReceive(Context context, Intent intent) {
+            assert BuildConfig.isInternetAvailable;
```

Comment:
> Maybe it's worth a more robust internet check?

If this matters we could have a separate feature request, but it seems relatively low-priority.